app/Jobs/UserRatePackage.php

```diff
@@ -60,7 +61,7 @@ public function handle()
 
     private function isSelfAuthored($package)
     {
```

Comment:
Is this going to add a lot of load, or are we only calling it on the package detail page?

Reply:
Looks like it will only be hit when we're submitting a rating.

The `RatingsController` calls `auth()->user()->ratePackage()` which dispatches this job.

Reply:
Thanks! (If you can't tell I was doing that from a phone 😬 didn't even check that it's on the damn job class. My bad.